### ISSUE TITLE
Using the Johns Hopkins database

### ISSUE THREAD

**Issue by mahyar-osn** [ABI-Covid-19/moh-data#9]
Switching to the [Johns Hopkins database](https://github.com/CSSEGISandData/COVID-19) as there is more info available (e.g. # recovered and # deaths etc.). Also, the format seems to be more consistent hence avoiding constant changes to the code.

EDIT: We still need to keep MoH in the code to fetch the probable and confirmed cases.

**Comment by agarny** [comment]
Just a reminder, we need/want data for:
- Cumulative confirmed cases;
- Cumulative probable cases;
- Cumulative number of deaths; and
- Cumulative number of recovered cases.

**Comment by mahyar-osn** [comment]
We can fetch these from Johns Hopkins by the look of it. Hopefully, I will have a new update to the code by the end of today to test with seir.

**Comment by agarny** [comment]
From what I can tell, the JHU data doesn't distinguish between confirmed and probable cases. In fact, confirmed+probable = confirmed for them...

**Comment by mahyar-osn** [comment]
You're right. I just went through the other csv files and seems like only confirmed is available. Thus will use JHU for the cumulative number of deaths and cumulative number of recovered cases + MoH for probable and confirmed cases (and also to confirm the validity of JHU in case if there were discrepancies).

**Comment by mahyar-osn** [comment]
Just edited the issue.

**Comment by agarny** [comment]
Otherwise, and FWIW, I have been populating a Google Sheets since the beginning of Covid-19 in NZ: https://docs.google.com/spreadsheets/d/16UMnHbnBHju-fK45aSdaJhVmrXJpy71oxSiN_AvqV84. There is a way to access that data from Python (see https://towardsdatascience.com/accessing-google-spreadsheet-data-using-python-90a5bc214fd2) and I have started implementing it, but it's probably going to be only for me to test things.

**Comment by agarny** [comment]
> Otherwise, and FWIW, I have been populating a Google Sheets since the beginning of Covid-19 in NZ: https://docs.google.com/spreadsheets/d/16UMnHbnBHju-fK45aSdaJhVmrXJpy71oxSiN_AvqV84. There is a way to access that data from Python (see https://towardsdatascience.com/accessing-google-spreadsheet-data-using-python-90a5bc214fd2) and I have started implementing it, but it's probably going to be only for me to test things.

Actually, scrap that. This requires authentication and although I got it working, it needs on a `client_secret.json` file, which we shouldn't really share publically. This being said, my Google Sheets is accessible to anyone who has the URL, so we can simply get the data using:
```python
import requests
response = requests.get(url='https://docs.google.com/spreadsheets/u/1/d/16UMnHbnBHju-fK45aSdaJhVmrXJpy71oxSiN_AvqV84/export?format=csv&id=16UMnHbnBHju-fK45aSdaJhVmrXJpy71oxSiN_AvqV84&gid=0')
print(response.content)
```

**Comment by mahyar-osn** [comment]
Thanks @agarny. I would also like to add that spreadsheet as an extra source. 

Meanwhile, I created a new PR related to the current issues.

**Comment by agarny** [comment]
> Thanks @agarny. I would also like to add that spreadsheet as an extra source.

I intend to update that Google Sheets as we go, so sure feel free to use as an extra source.

> Meanwhile, I created a new PR related to the current issues.

Thanks, am going to check it out.
